src/nyc_trees/sass/partials/_map.scss

```diff
@@ -15,6 +15,9 @@
 }
 
 .map-container {
+  transition: bottom 400ms;
+  -ms-transition: bottom 400ms;
+  -webkit-transition: bottom 400ms;
```

Comment:
I don't think it's crucial that you remove these for this PR, but as an FYI, there is no need to include these prefixed selectors, since we now run our CSS files through autoprefixer